packages/core/src/fst/actor.ts

```diff
@@ -0,0 +1,36 @@
+import { FST } from '.';
+import { ActorRef, EventObject, Observer } from '..';
+import { toObserver } from '../utils';
+
+export function toActorFST<TState, TEvent extends EventObject, TOutput = any>(
```

Comment:
`toActorFST` sounds like a function to convert to some kind of an FST actor, or converting the actor to an FST whereas this converts FST to an actor, maybe `fstToActor` would fit this better? or just `toActor` in the FST-related package/directory? it could be used like this:
```
import * as FST from '...'
FST.toActor(/**/)
```

Reply:
Good call, will rename.